src/extract.rs

```diff
@@ -50,31 +50,22 @@ pub struct ExtractedLightOccluder2d {
 
 pub fn extract_light_occluders(
     mut commands: Commands,
-    mut previous_len: Local<usize>,
     light_occluders_query: Extract<
-        Query<(Entity, &LightOccluder2d, &GlobalTransform, &ViewVisibility)>,
+        Query<(RenderEntity, &LightOccluder2d, &GlobalTransform, &ViewVisibility)>,
     >,
 ) {
     commands.spawn(ExtractedLightOccluder2d::default());
```

Comment:
This is adding an empty `ExtractedLightOccluder2d` that is not needed any more. In fact if you run the example you'll find an artifact in the middle of the screen that is a ghost occluder. The loop in `sdf.wgsl:27` as it is accounts for this empty element and starts at element `2u` (I cannot recall why 2). It should start from `0u` from now on.

My bad that this is poorly commented. It's related to how GPU buffers are created in bevy.